noisemaker/src/Noisey.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import Sound from 'react-sound';
+import { Container, Row, Col, Button } from 'reactstrap';
+
+const sounds = new Audio("bensound-summer.mp3");
+
+
+class Noisey extends Component {
+    constructor(props) {
+        super(props);
+        
+    }
+    soundbite() {
+        const sound = document.getElementsByClassName('audio-element')[0]
+        sound.play()
+    }
+    soundbite1() {
+        const sound = document.getElementsByClassName('audio-element')[1]
+        sound.play()
```

Comment:
if you are getting specific elements why not just have them have their own id instead of using the same class name?